schema/me/update_conversation.js

```diff
@@ -15,20 +16,27 @@ export default mutationWithClientMutationId({
     buyer_outcome: {
       type: new GraphQLNonNull(BuyerOutcomeTypes),
     },
-    id: {
-      type: new GraphQLNonNull(GraphQLString),
+    ids: {
+      type: new GraphQLList(GraphQLString),
     },
   },
-  outputFields: ConversationFields,
-  mutateAndGetPayload: ({ buyer_outcome, id }, request, { rootValue: { accessToken } }) => {
+  outputFields: {
+    conversations: {
+      type: new GraphQLList(ConversationType),
+      resolve: (conversations) => conversations,
```

Comment:
Normally with GraphQL-js you don't need to add a resolver if the field from the payload is named exactly the same as the field definition. Not sure it applies to mutations too, but I wouldn't be surprised.